### ISSUE TITLE
Grammar Mistake "Magazine Cutout"

### ISSUE THREAD

**Issue by Thretcha** [exercism/rust#1305]
### Introduction of the concept magazine-cutout:
"Rust's entry API provides a view into a single entry in map, which may be either a HashMap or a BTreeMap. The entry may be either occupied **and** vacant, and the API provides methods to modify the contents of the entry."

should probably be:

"Rust's entry API provides a view into a single entry in map, which may be either a HashMap or a BTreeMap. The entry may be either occupied **or** vacant, and the API provides methods to modify the contents of the entry."

_TLDR: replace "occupied and vacant" with "occupied or vacant"_

Sorry for opening an issue instead of a pull request, but I was unsure on how to properly do a pull request with a fix. (Attempting to do so without being certain that I don't mess something up, is kind of scary on a public repo)

**Comment by coriolinus** [comment]
Closed by https://github.com/exercism/rust/pull/1311.
